ci-operator/step-registry/addon/aws/osd/addon-aws-osd-workflow.yaml

```diff
@@ -0,0 +1,30 @@
+workflow:
+  as: addon-aws-osd
+  steps:
+    env:
+      API_HOST: stage
+      CHANNEL_GROUP: candidate
+      COMPUTE_MACHINE_TYPE: "m5.4xlarge"
+      MACHINE_POOL_INSTANCE_TYPE: "m5.2xlarge"
```

Comment:
why are these 2 needed?

Reply:
removed cluster infra settings from all workflows.
we will provide that from outside as the scenario requires.